src/components/LandingPage/ClientOnly.tsx

```diff
@@ -0,0 +1,20 @@
+import { useEffect, useState } from "react";
+
+type OnlyClientProps = {
+  children: React.ReactNode;
+};
+
+const ClientOnly: React.FC<OnlyClientProps> = ({ children }) => {
+  const [hasMounted, setHasMounted] = useState(false);
+
+  useEffect(() => {
+    setHasMounted(true);
+  }, []);
+
+  if (!hasMounted) {
+    return null;
+  }
+  return <>{children}</>;
```

Comment:
```suggestion
  return <>{ hasMounted  && children}</>;
```
teste isso pra ver se funciona